docs/content/latest/explore/follower-reads/fr-ysql.md

```diff
@@ -15,15 +15,19 @@ showAsideToc: true
 ---
```

Comment:
Add Beta tag/icon?

Reply:
I'm not sure in this case. We usually add a beta tag when there's content.

Reply:
Let's see if somebody complains.